Source/gui/SNTBinaryMessageWindowView.swift

```diff
@@ -0,0 +1,313 @@
+          .foregroundColor(.white)
+          .cornerRadius(40)
+          .scaleEffect(configuration.isPressed ? 0.8 : 0.9)
+  }
+}
+
+struct SNTBinaryMessageEventView: View {
+  let e: SNTStoredEvent? 
+  let customURL: NSString?
+
+  @State private var isShowingDetails = false
+
+  var body: some View {
+    Spacer()
+
+    HStack(spacing: 20.0) {
+      VStack(alignment:.trailing, spacing:10.0) {
+        if e?.needsBundleHash ?? false {
+          Text("Bundle Hash")
+        }
+
+        if e?.fileBundleName != "" {
+          Text("Application").bold().font(Font.system(size:12.0))
+        } else if e?.filePath != "" {
+          Text("Filename").bold().font(Font.system(size:12.0))
+        }
+
+        if Publisher(e?.signingChain, e?.teamID) != "" {
+          Text("Publisher").bold().font(Font.system(size:12.0))
+        }
+
+        Text("User").bold().font(Font.system(size:12.0))
+      }
+
+      Divider()
+
+      VStack(alignment:.leading, spacing:10.0) {
+        if e?.needsBundleHash ?? false {
+          // TODO: Implement bundle hashing
+          ProgressView()
+
+          /*
+            To be implemented in the near future...
+
+            if (!self.event.needsBundleHash) {
+              [self.bundleHashLabel removeFromSuperview];
+              [self.hashingIndicator removeFromSuperview];
+              [self.foundFileCountLabel removeFromSuperview];
+            } else {
+              self.openEventButton.enabled = NO;
+              self.hashingIndicator.indeterminate = YES;
+              [self.hashingIndicator startAnimation:self];
+              self.bundleHashLabel.hidden = YES;
+              self.foundFileCountLabel.stringValue = @"";
+            }
+          */
+        }
+
+        if let bundleName = e?.fileBundleName {
+          Text(bundleName).textSelection(.enabled)
+        } else if let filePath = e?.filePath {
+          Text((filePath as NSString).lastPathComponent).textSelection(.enabled)
+        }
+
+        if let publisher = Publisher(e?.signingChain, e?.teamID) {
+          Text(publisher).textSelection(.enabled)
+        }
+
+        Text(e?.executingUser ?? "").textSelection(.enabled)
+      }
+    }.sheet(isPresented: $isShowingDetails) {
+      SNTBinaryMessageEventExpandedView(e: e)
+    }
+
+    HStack {
+      Button(action: { isShowingDetails = true}) {
+        HStack(spacing:2.0) {
+          Text("\(Text("M").underline())ore Details").foregroundColor(.blue)
+          Image(systemName:"info.circle").foregroundColor(.blue)
+        }
+      }
+      .buttonStyle(ScalingButtonStyle())
+      .keyboardShortcut("m", modifiers: .command)
+
+      Button(action: copyDetailsToClipboard) {
+        HStack(spacing:2.0) {
+          Text("\(Text("C").underline())opy Details").foregroundColor(.blue)
+          Image(systemName:"pencil.and.list.clipboard").foregroundColor(.blue)
+        }
+      }
+      .buttonStyle(ScalingButtonStyle())
+      .keyboardShortcut("c", modifiers: .command)
```

Comment:
Does Cmd+C shortcut still work when the detail sheet is displayed? And can you highlight text from the sheet? If so, I'm wondering what happens, does just the highlighted text get copied or does the whole sheet content get copied?

Reply:
> Does Cmd+C shortcut still work when the detail sheet is displayed?
No

> And can you highlight text from the sheet? 
Yes

> If so, I'm wondering what happens, does just the highlighted text get copied or does the whole sheet content get copied?
You can copy the highlighted text.

Now that you've pointed it out, overloading Cmd+C like this feels wrong, so I've updated to use Cmd+D instead. The answers above still apply.